### ISSUE TITLE
Disable Backup Sync

### ISSUE THREAD

**Issue by FredericJacobs** [anyproto/anytype-ts#122]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

I would like to be able to use AnyType without the backup sync. I have not found a way to do that.

### Suggested solution

Include a setting to allow disabling the use of the backup node.

I don't mind paying to not use any ressources! :)

**Comment by fenilgmehta** [comment]
Both solution (i.e. self-hosting configuration & firewall) are quite complex. Can we please have a simple checkbox/switch to control this.

**Comment by fuksman** [comment]
@fenilgmehta yes, we will deliver it in the upcoming releases, along with simplification of self-hosting.

**Comment by fenilgmehta** [comment]
@fuksman thanks a lot 🙏
Since this issue is closed, is there any other place (e.g. GitHub issue or some link) which we can use to check the status?
Is it planned for this year?

**Comment by fuksman** [comment]
We're also working on mirroring the tasks from our internal tracker to the GitHub project. Until we implement this, I'm opening the issue to make it possible to check the status.

And yes, these features are planned for this year. Stay tuned 🙌
